### ISSUE TITLE
Enableing mod kills game entirely

### ISSUE THREAD

**Issue by Alatarius** [joe7575/techage_modepack#18]
I started a game with this mod only and I get this error:

2023-10-14 21:32:22: ERROR[Main]: ModError: Failed to load and run script from D:\Games\Minetest\bin\..\mods\techage_modpack\ta4_jetpack\init.lua:
2023-10-14 21:32:22: ERROR[Main]: ...st\bin\..\mods\techage_modpack\ta4_jetpack/manual_DE.lua:1: attempt to call field 'add_to_manual' (a nil value)
2023-10-14 21:32:22: ERROR[Main]: stack traceback:
2023-10-14 21:32:22: ERROR[Main]: 	...st\bin\..\mods\techage_modpack\ta4_jetpack/manual_DE.lua:1: in main chunk
2023-10-14 21:32:22: ERROR[Main]: 	[C]: in function 'dofile'
2023-10-14 21:32:22: ERROR[Main]: 	...inetest\bin\..\mods\techage_modpack\ta4_jetpack\init.lua:551: in main chunk
2023-10-14 21:32:22: ERROR[Main]: Check debug.txt for details.
2023-10-14 21:32:22: ACTION[Main]: Server: Shutting down

**Comment by HMS-jost** [comment]
Check whether all mods in the modpack are activated
![Bildschirmfoto_2023-10-15_17-54-38](https://github.com/joe7575/techage_modpack/assets/106076945/9431a197-0c62-4d8a-99c6-f459edbf210f)

**Comment by joe7575** [comment]
Check whether all mods in the modpack are activated
![Bildschirmfoto_2023-10-15_17-54-38](https://github.com/joe7575/techage_modpack/assets/17710938/e969b425-d5ce-4fdf-b47b-41041607d508)

**Comment by Alatarius** [comment]
Yep, they are all on as this is the only mod active on this world

**Comment by joe7575** [comment]
> as this is the only mod active on this world

Then this is your problem. According to the [README](https://github.com/joe7575/techage_modpack#further-dependencies) you need 3d_armor in addition.
